analysis/callGraph.ml

```diff
@@ -65,22 +65,33 @@ let create ~environment ~source =
                   };
                 _;
               } ->
-                add_call_edge call_graph caller callee
+                Some callee
             | _ ->
-                call_graph
+                None
           in
           Annotated.Access.create access
-          |> Annotated.Access.fold ~resolution ~initial:call_graph ~f:add_call_edge
+          |> Annotated.Access.fold ~resolution ~initial:None ~f:resolve_access
```

Comment:
@dkgi, basically, I noticed that `Annotated.Access.fold` doesn't create a `Signature` for some types of calls (see test). Not sure if this is "as intended" or if there is a missing case that should be added in `Access.fold`. For now, the logic below will catch missing edges.

Reply:
Not sure I see what you mean in the tests. Are we missing those just in conditional tests? If so, that sounds wrong.

Sounds like this should be fixed in the access fold. Let me know how you want to proceed. Fine with landing this and adding a TODO.

Reply:
No it's not specific to conditional test (so I've removed that). It's just a minimal example of where call signatures are not created. 

> Sounds like this should be fixed in the access fold.

Possibly right? As a very simple example:

```python
def foo():
    bar()
```

When I fold over the `bar` access and print each element (there's only one), I see the resolution is only:

```
DUMP element: (AnnotatedAccess.Element.Attribute
   { AnnotatedAccess.Element.attribute = bar;
     origin = (AnnotatedAccess.Element.Module ); defined = false })
```

> Let me know how you want to proceed. Fine with landing this and adding a TODO.

Think this is fine for now until you have a chance to look into whether this is intended. Either way, the tests are good. Apparently you can edit my PR (e.g., add a TODO with task number), but I have put a placeholder TODO anyway.